src/PlotRecipes/Potentials.jl

```diff
@@ -58,29 +58,33 @@ function get_crosssection_idx_and_value(grid::CylindricalGrid{T}, r::Union{Real,
     cross_section, idx, value, units
 end
 
-@recipe function f(sp::ScalarPotential{T,3,Cylindrical}; r = missing, φ = missing, z = missing, contours_equal_potential = false, full_det = false) where {T <: SSDFloat}
+@recipe function f(sp::ScalarPotential{T,3,Cylindrical}; r = missing, φ = missing, z = missing, contours_equal_potential = false, full_det = false, all_point_types_bits = false) where {T <: SSDFloat}
 
     gradient::Symbol, clims::Tuple{RealQuantity, RealQuantity}, name::String, punit::Unitful.Units = _get_potential_plot_information(sp)
 
     if !(sp.grid[2][end] - sp.grid[2][1] ≈ 2π) sp = get_2π_potential(sp, n_points_in_φ = 72) end
 
     grid::CylindricalGrid{T} = sp.grid
     cross_section::Symbol, idx::Int, value::T, units::Unitful.Units = get_crosssection_idx_and_value(grid, r, φ, z)
+
+    if all_point_types_bits && eltype(sp.data) == PointType && !haskey(plotattributes, :clims)
+         clims = (minimum(sp.data), maximum(sp.data))
+    end
     
     title --> name * " @ $(cross_section) = $(round(units, Float64(uconvert(units,value*(cross_section == :φ ? u"°" : internal_length_unit))), sigdigits=3))"
     colorbar_title --> name
     seriescolor --> gradient
     clims --> clims
     unitformat --> :slash
 
-    sp, cross_section, idx, value, punit, contours_equal_potential, full_det
+    sp, cross_section, idx, value, punit, contours_equal_potential, full_det, all_point_types_bits
```

Comment:
... then you can pass this down as attribute (I think), and don't necessarily need to add this for `ElectricField`
```suggestion
    all_point_types_bits --> all_point_types_bits
    
    sp, cross_section, idx, value, punit, contours_equal_potential, full_det
```